lib/iron_bank/configuration.rb

```diff
@@ -78,5 +81,21 @@ def credentials?
     def retry_options
       @retry_options ||= IronBank::Client::DEFAULT_RETRY_OPTIONS
     end
+
+    def excluded_fields
+      return {} unless excluded_fields_file
+
+      unless File.exist?(excluded_fields_file)
+        IronBank.logger.warn "Cannot open #{excluded_fields_file}"
```

Comment:
maybe warn or info that the file is not present instead of unable to open? (the latter feels/sounds like an error condition)